src/stac_api_validator/validations.py

```diff
@@ -3199,23 +3199,24 @@ def validate_item_search_ids(
         r_session=r_session,
     )
 
-    items = body.get("features")  # type: ignore
-    if items and len(items) >= 2:
-        _validate_search_ids_with_ids(
-            search_url, [items[0].get("id")], methods, errors, r_session
-        )
-        _validate_search_ids_with_ids(
-            search_url,
-            [items[0].get("id"), items[1].get("id")],
-            methods,
-            errors,
-            r_session,
-        )
-        _validate_search_ids_with_ids(
-            search_url, [i["id"] for i in items], methods, errors, r_session
-        )
-    else:
-        warnings += f"[{Context.ITEM_SEARCH}] GET Search with no parameters returned < 2 results"
+    if body:
+        items = body.get("features")  # type: ignore
+        if items and len(items) >= 2:
+            _validate_search_ids_with_ids(
+                search_url, [items[0].get("id")], methods, errors, r_session
+            )
+            _validate_search_ids_with_ids(
+                search_url,
+                [items[0].get("id"), items[1].get("id")],
+                methods,
+                errors,
+                r_session,
+            )
+            _validate_search_ids_with_ids(
+                search_url, [i["id"] for i in items], methods, errors, r_session
+            )
+        else:
+            warnings += f"[{Context.ITEM_SEARCH}] GET Search with no parameters returned < 2 results"
```

Comment:
Feels like we should add an error if the search returns no body, since that's incorrect behavior?

Reply:
Done :)